src/components/Table/Table.js

```diff
@@ -0,0 +1,56 @@
+import { IoIosArchive } from "react-icons/io";
+import { IoMdCreate } from "react-icons/io";
+import { BsArrowDownUp } from "react-icons/bs";
+import './Table.css';
+
+const Table = () => {
+    return (
+        <table className='table'>
+            <thead>
+                <tr id='first'>
+                    <th>ID<BsArrowDownUp id='icon' /></th>
+                    <th>Category<BsArrowDownUp id='icon' /></th>
+                    <th>Name<BsArrowDownUp id='icon' /></th>
+                    <th>Quantity<BsArrowDownUp id='icon' /></th>
+                    <th>Price (₴)<BsArrowDownUp id='icon' /></th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody id='second'>
```

Comment:
It is bad practice to write same components. Read the topic about .map() - https://www.pluralsight.com/guides/how-to-use-the-map()-function-to-export-javascript-in-react.

If you can, rewrite this way